tests/integration_tests/test_ingest_component.py

```diff
@@ -99,6 +99,145 @@ def wrong_format_test_file() -> BinaryIO:
         yield file
 
 
+@pytest.fixture(scope="function")
+def pathfinders_round_1_file_success() -> BinaryIO:
+    """An example spreadsheet for reporting round 4 of Towns Fund that should ingest without validation errors."""
+    with open(Path(__file__).parent / "mock_pf_returns" / "PF_Round_1_Success.xlsx", "rb") as file:
+        yield file
+
+
+@pytest.fixture(scope="function")
```

Comment:
I think we can make both of these fixtures `module` scope, as we'll only be reading the return value anyway

Reply:
I noticed I'm getting intermittent 'read of closed file' errors during testing with these example spreadsheets when scop is set to moduke, and it varies between each test run whether it succeeds or not. It seems to pass consistently when set to function scope - I'm not sure why this is happening though

Reply:
The pattern I suggested doesn't work with `module` scope fixtures because the file closes after one test has finished, which causes subsequent tests to fail. Resolving this comment.